client/src/public/ts/toast-notification.ts

```
@@ -0,0 +1,127 @@
+enum ToastNotificationPosition {
+  TOP_RIGHT = 2,
+  BOTTOM_RIGHT = 8
+}
+
+class ToastNotification {
+  static notifications: Array<ToastNotification> = [];
+  static nextID: number = 1;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)